konig-schemagen/src/main/java/io/konig/schemagen/gcp/SpannerTable.java

```diff
@@ -0,0 +1,120 @@
+package io.konig.schemagen.gcp;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openrdf.model.URI;
+
+import io.konig.activity.Activity;
+import io.konig.gcp.datasource.SpannerTableReference;
+import io.konig.shacl.Shape;
+
+public class SpannerTable {

```

Comment:
Need your help in defining SHACL constructs for constraints like Primary Key, Unique Key

Can we use sh:targetNode for interleaving tables in a cluster?